openaq_api/openaq_api/v3/routers/providers.py

```diff
@@ -103,7 +103,6 @@ async def fetch_providers(query, db):
     , datetime_last
     , datetime_added
     , measurements_count
-    , locations_count
     , countries_count
```

Comment:
I think this field and `measurementsCount` should also go if we're removing locations count

Reply:
@majesticio still need to remove `countries_count`